src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -191,6 +191,24 @@ studio.security.ldap.userAttribute.siteId: crafterSite
 studio.security.ldap.userAttribute.groupName: crafterGroup
 # Global admin group
 studio.security.global.adminGroup: crafter-admin
+# Authentication via headers enabled
+studio.authentication.headers.enabled: false
+# Authentication header for secure key
```

Comment:
Change the comment to indicate this is the `key header name`, what header to look for